UnitsNet.WindowsRuntimeComponent/GeneratedCode/UnitAbbreviationsCache.g.cs

```diff
@@ -212,17 +212,17 @@ public sealed partial class UnitAbbreviationsCache
                 ("en-US", typeof(DurationUnit), (int)DurationUnit.Hour, new string[]{"h", "hr", "hrs", "hour", "hours"}),
                 ("ru-RU", typeof(DurationUnit), (int)DurationUnit.Hour, new string[]{"ч"}),
                 ("en-US", typeof(DurationUnit), (int)DurationUnit.Microsecond, new string[]{"µs"}),
-                ("ru-RU", typeof(DurationUnit), (int)DurationUnit.Microsecond, new string[]{"мкс"}),
+                ("ru-RU", typeof(DurationUnit), (int)DurationUnit.Microsecond, new string[]{"мкс мксек"}),
```

Comment:
Hmm, this is strange. Why did the AppVeyor checks pass the first time? Shouldn't have my tests failed? 
Code generated looks fine now, but I don't know- how do you really make sure..

Reply:
AppVeyor regenerates the code too, since often when merging multiple PRs the committed generated code is out-of-date.

Reply:
Yes, but I guess it doesn't run the same tests? That's a shame, cause it would have also caught [this](https://github.com/angularsen/UnitsNet/blob/5e26a161615fb2873923c72bc41318c6b1230e81/UnitsNet.WindowsRuntimeComponent/CustomCode/UnitParser.cs#L38) :)

Reply:
It should run the exact same tests as when running `build.bat` locally - which to my knowledge should run all the tests in the solution. Isn't that the case?

I didn't quite get what you meant by the linked "this" code snippet, what should it have caught there?

Reply:
Oh, you mean it doesn't run the tests for WRC? Yes that's right, we didn't put in the effort to duplicate the tests for WRC.

Reply:
The snippet points to the same issue from #652 (that you fixed in #655), but for WRC.
As a side-note, I was never actually able to get my build to complete using the batch files.

Reply:
Erg, it won't let me attach the build log, but here is the highlight:
```
Restore completed in 26,38 ms for C:\Users\galin\Documents\GitHub\ConcentrationUnits\UnitsNet\UnitsNet.Serialization.JsonNet.Tests\UnitsNet.Serialization.JsonNet.Tests.csproj.
  Retrying 'FindPackagesByIdAsyncCore' for source 'https://camo.githubusercontent.com/da308986135cea0abf9a027dc6d147d397d5e383/68747470733a2f2f696d672e736869656c64732e696f2f6e756765742f767072652f6f72636865737472612e636f72652e737667/FindPackagesById()?id='UnitsNet.Serialization.JsonNet'&semVerLevel=2.0.0'.
  Response status code does not indicate success: 404 (Not Found).
  Retrying 'FindPackagesByIdAsyncCore' for source 'https://camo.githubusercontent.com/da308986135cea0abf9a027dc6d147d397d5e383/68747470733a2f2f696d672e736869656c64732e696f2f6e756765742f767072652f6f72636865737472612e636f72652e737667/FindPackagesById()?id='UnitsNet.Serialization.JsonNet'&semVerLevel=2.0.0'.
  Response status code does not indicate success: 404 (Not Found).
C:\Program Files\dotnet\sdk\2.1.505\NuGet.targets(114,5): error : Failed to retrieve information about 'UnitsNet.Serialization.JsonNet' from remote source 'https://camo.githubusercontent.com/da308986135cea0abf9a027dc6d147d397d5e383/68747470733a2f2f696d672e736869656c64732e696f2f6e756765742f767072652f6f72636865737472612e636f72652e737667/FindPackagesById()?id='UnitsNet.Serialization.JsonNet'&semVerLevel=2.0.0'. [C:\Users\galin\Documents\GitHub\ConcentrationUnits\UnitsNet\UnitsNet.sln]
C:\Program Files\dotnet\sdk\2.1.505\NuGet.targets(114,5): error :   Response status code does not indicate success: 404 (Not Found). [C:\Users\galin\Documents\GitHub\ConcentrationUnits\UnitsNet\UnitsNet.sln]

Build FAILED.
```

Reply:
Weird, I have never seen this before. I initially suspected something related to SourceLink, but googling that hostname gives me this:
https://help.github.com/en/articles/about-anonymized-image-urls

Googling some more on 404 and FindPackagesByIdAsyncCore, it seems that maybe you have multiple nuget sources configured on your machine? Maybe you have added a myget source or similar?

https://github.com/NuGet/Home/issues/6140
https://github.com/dotnet/cli/issues/6680

Seems like a bug with `dotnet restore` that is not yet fixed.

Reply:
Nice, disabling the alpha-channel source fixed the issue(I still got to keep the other sources so I guess it's kinda random). Anyway, I feel much better now. Thanks!

Reply:
I think the order matters, one of the links said it would only try the *first* source and ignore the others. Try moving the official nuget source as the first one.

Reply:
Yes, saw that- but for me it was already the last source. Anyway, I don't need it anymore.